recipes-browser/cog/cog_0.12.0.bb

```diff
@@ -0,0 +1,17 @@
+require cog.inc
+require conf/include/devupstream.inc
+
+SRC_URI[sha256sum] = "aecf546d7b0645119d79559c5574cb5eb68364fc8409dfbd47a4920bd1f221bc"
+
+# Avalilable from 0.10.X
+PACKAGECONFIG[headless] = "-DCOG_PLATFORM_HEADLESS=ON,-DCOG_PLATFORM_HEADLESS=OFF,wpebackend-fdo"
+
+# Available from 0.12.X (future) or for devupstream (HEAD:master. Ref: https://github.com/Igalia/cog/commit/b595d12945148205af9da765cf05d092cbc234c4)
+PACKAGECONFIG[wl] = "-DCOG_PLATFORM_WL=ON,-DCOG_PLATFORM_WL=OFF,wpebackend-fdo"
```

Comment:
I suppose we can also remove this line, now that we will remove the `inherit devupstream` from the 0.10.x recipe, right?

Reply:
In this case we should also remove the 0.6.0 cog recipe. Actually I think it should not be there since a while According with the compatibility matrix in https://wpewebkit.org/release/schedule/ the Cog 0.6 is compatible with 2.30 what is not longer an available version of wpe in this branch:

WPE WebKit | libwpe | WPEBackend-fdo | Cog
-- | -- | -- | --
2.34.x | 1.12.x, 1.10.x, 1.8.x | 1.12.x, 1.10.x | 0.8.x, 0.10.x, 0.12.x
2.32.x | 1.10.x, 1.8.x, 1.6.x | 1.10.x, 1.8.x | 0.8.x, 0.10.x
2.30.x | 1.8.x, 1.6.x | 1.10.x, 1.8.x | 0.8.x, **0.6.x**

So I'd suggest to remove the cog_0.6.0 recipe then move this flag to `cog.inc`:

```
# Available from 0.12.X (future) or for devupstream (HEAD:master. Ref: https://github.com/Igalia/cog/commit/b595d12945148205af9da765cf05d092cbc234c4)
PACKAGECONFIG[wl] = "-DCOG_PLATFORM_WL=ON,-DCOG_PLATFORM_WL=OFF,wpebackend-fdo"
```
... and finally remove the old deprecated `fdo` flag:

# TODO: (deprecated) Last cog version using this flag is 0.10.X
PACKAGECONFIG[fdo] = "-DCOG_PLATFORM_FDO=ON,-DCOG_PLATFORM_FDO=OFF,wpebackend-fdo"